lambdas/db-migration/package.json

```diff
@@ -13,7 +13,7 @@
     "clean": "rm -rf dist",
     "db:local:schema": "env PGPASSWORD=password pg_dump -h localhost -U postgres --schema-only --no-owner | egrep -v '^--' | cat -s | tee schema.sql",
     "webpack": "../../node_modules/.bin/webpack",
-    "build-lambda-zip": "(cd dist/webpack && cp -R ../lambda/migrations . && node ../../../../bin/zip.js lambda.zip index.js migrations/*.js)",
+    "build-lambda-zip": "(cd dist/webpack && node ../../../../bin/zip.js lambda.zip index.js)",
```

Comment:
This is the removal of the copy of the migrations directory and it's contents from the build of db-migration-lambda.